### PR TITLE
Changes random human generators to use natural colors for hair.

### PR DESCRIPTION
**Pull request by gyurka66**
Added a list of common natural hair colors and changed random human generators to use it.

This is good because players will encounter Chernobyl refugees less often

## About The Pull Request
Added a new global list to mobs.dm in _HELPERS of hair colors and added /proc/random_hair_color() to access it.
Changed /proc/randomize_human() in create_mob.dm,
/datum/preferences/proc/process_link() in preferences.dm,
/obj/item/clothing/head/wig/random/Initialize() in misc_special.dm,
/datum/preferences/proc/random_character() in preferences_setup.dm to use  random_hair_color() instead of random_short_color.

## Why It's Good For The Game
This is good because players will encounter Chernobyl refugees less often when dealing with cloning, ruin exploration or just the random body setting in the character creator.

## Changelog
:cl: Gyurka66
tweak: Tweaked random human generators to use more natural hair colors.
/:cl:

### CODE REVIEW

**Comment by Fikou** [comment]
shouldnt it also make facial hair color the same as hair color to not look weird

**Comment by gyurka66** [comment]
> 
> 
> shouldnt it also make facial hair color the same as hair color to not look weird

That's already the case

**Comment by qwertyquerty** [comment]
I think the list should have a lot more in-between colors

**Comment by MCterra10** [comment]
make it a toggle bruh

**Comment by gyurka66** [comment]
> 
> 
> make it a toggle bruh

What do you mean? on the character creator window?

**Comment by MCterra10** [comment]
> 
> 
> > make it a toggle bruh
> 
> What do you mean? on the character creator window?

Make it toggle to have natural random hair colours.
Some people might like their neon hair

**Comment by Pill-U** [comment]
Green hair gang disapproves of this pr

File: code/__HELPERS/mobs.dm [hunk 1]
```diff
@@ -170,6 +170,30 @@ GLOBAL_LIST_INIT(skin_tones, list(
 	"african2"
 	))
 
+/proc/random_hair_color()
+	return pick(GLOB.hair_colors)
+
+GLOBAL_LIST_INIT(hair_colors, list(
```
Comment (ike709) [inline code/__HELPERS/mobs.dm:176]: Don't make this a global list, just make it a list inside the proc's pick()

**Pull request closed without merging**